Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL = /bin/bash
 
 build_tag ?= dul-arclight
-builder_image ?= gitlab-registry.oit.duke.edu/devops/containers/ruby:2.6-main
+builder_image ?= gitlab-registry.oit.duke.edu/devops/containers/ruby:2.7-main
```

Comment:
actually I think we can just delete this Makefile. I can take on the small task of "rip out Duke's build stuff we aren't using any more and update the README(s) accordingly"

Reply:
Sounds good to me.